lib/index.ts

```diff
@@ -1,8 +1,8 @@
-import API from 'api/index';
-import bootstrapUI from 'ui/index';
+import API from './api/index';
+import bootstrapUI from './ui/index';
```

Comment:
I don't know, the TS plugin for Sublime says `Cannot find module 'api/index'` even though it does. Is there a setting to tell TS the resolve root, similar to Webpack?